packages/blade/src/components/SideNav/types.ts

```diff
@@ -42,6 +51,13 @@ type SideNavLinkProps = {
    */
   title: string;
 
+  /**
+   * description of the Link
+   *
+   * **Note**: Only applicable for L2 items
+   */
+  description?: string;
```

Comment:
can we have a check for this ? we can throw warning if description is provided to L2 items ?

Reply:
makes sense. We can throw dev-only error. I was avoiding production error because if backend API mistakenly sends description then we don't want the sidenav to error out completely.